src/server/multi_command_squasher.cc

```diff
@@ -205,7 +203,6 @@ OpStatus MultiCommandSquasher::SquashedHopCb(EngineShard* es, RespVersion resp_v
     CheckConnStateClean(local_state);
   }
 
-  reverse(sinfo.replies.begin(), sinfo.replies.end());
```

Comment:
Wasn't there a reason we reversed this ?

Reply:
of course there was. It was convenient to pull the reply later when we reconstructed the connection replies.
I've adjusted the code:
https://github.com/dragonflydb/dragonfly/pull/5011/files#diff-a781086bea40e29c13352594fead4fed8c2bb2f4529a8d3058a8a6dd41e264dcR259

Reply:
:ship: :it: